### DISCUSSION TITLE
Strapi doesn't delete content-type field

### DISCUSSION THREAD

**Discussion by szerwinski** [strapi/strapi#8256] [Legacy Topics]
When making api requests, for example a POST request to an endpoint named "new-restaurant", the old fields (that was already removed) still show up in these new entries. 

Example: POST request to myrestaurants/new-restaurant
{
id: "restaurant id",
name: "new restaurant name",
zip code: "restaurant zip code",
}

API response:

{
id: "restaurant id",
name: "new restaurant name",
zip code: "restaurant zip code",
}

and when I delete, from the admin pannel, the entry "zip code" from this content-type, and make another request to the same endpoint but without the field that was deleted from this content-type, for example:

 POST request to myrestaurants/new-restaurant
{
id: "restaurant id",
name: "new restaurant name",
}

my response is:

{
id: "restaurant id",
name: "new restaurant name",
zip code: null,
}

I tried looking after all the files in my strapi projects, the models, schemas and all of it, and didn't find this deleted field mentioned anywhere, but it still comes out with my new POST requests, and I don't know why?!

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
